acceptance/Rakefile

```diff
@@ -9,9 +9,13 @@ ONE_DAY_IN_SECS = 24 * 60 * 60
 REPO_CONFIGS_DIR = "repo-configs"
 CLEAN.include('*.tar', REPO_CONFIGS_DIR, 'merged_options.rb')
 
+# If elsewhere we're depending on internal network resources
+# then assume we can depend on them here
+INTERNAL_NETWORK = ENV['GEM_SOURCE'] && ENV['GEM_SOURCE'] =~ /puppetlabs/
```

Comment:
My $.02: I'd prefer an environment variable per mirror. This solution makes me worry that 6 months from now someone will tweak `GEM_SOURCE` and then we'll all be scratching our heads about why all the other mirrors got turned off.